app/src/components/DataRequirements.tsx

```diff
@@ -0,0 +1,145 @@
+import { Accordion, Center, createStyles, em, getBreakpointValue, List, Paper, rem } from '@mantine/core';
+
+interface codeInterface {
```

Comment:
```suggestion
interface CodeInterface {
```
Nitpick, but it's best practice to capitalized interfaces in TypeScript - changing this will cause a few resulting changes throughout this file anywhere that `codeInterface` is used